LEGO1/lego/legoomni/src/build/legocarbuildpresenter.cpp

```diff
@@ -507,6 +507,44 @@ LegoTreeNode* LegoCarBuildAnimPresenter::FindNodeByName(LegoTreeNode* p_treeNode
 	return NULL;
 }
 
+// FUNCTION: BETA10 0x10073c20
+inline void Exchange(MxS16* p_value1, MxS16* p_value2)
+{
+	MxS16 temp = *p_value1;
+	*p_value1 = *p_value2;
+	*p_value2 = temp;
+}
```

Comment:
Is there a standard library or a known Mx function for that?

Reply:
Yes, try `Swap` in `legoutil.h`

Reply:
Good catch!